param/__init__.py

```diff
@@ -1426,6 +1430,15 @@ def __init__(self, default=None, **params):
         from numpy import ndarray
         super(Array,self).__init__(ndarray, allow_None=True, default=default, **params)
 
+    @classmethod
+    def serialize(cls, value):
+        return value.tolist()
+
+    @classmethod
+    def deserialize(cls, value):
+        from numpy import array
+        return array(value)
```

Comment:
Not super important but it would be nice if you could pass in an array without making a copy.

```suggestion
        from numpy import asarray
        return asarray(value)
```

Reply:
Good suggestion, thanks.